doc/nrf/libraries/dfu/dfu_target.rst

```diff
@@ -47,7 +71,7 @@ When the image data transfer is completed, the application using the DFU target
 2. Call the :c:func:`dfu_target_schedule_update` function to mark the firmware as *ready to be booted*.
    On the next reboot, the device will run with the new firmware.
 
-When the application has to collect another MCUboot image, it must either reset the DFU target using :c:func:`dfu_target_reset` or schedule the upgrade for the current image.
+When the application has to collect another MCUboot image, it must either erase the target area using the :c:func:`dfu_target_reset` function or schedule the upgrade for the current image.
```

Comment:
Why does it have to do this? The update will never go through, its it from a usability perspective?

Reply:
To be honest, I don't know what the intend of this clause was originally. It sounds like it tries to describe two different use cases:
* If the application needs to overwrite the fota area with a new image, it must call `dfu_target_reset()`
* If the application wants to download another MCUboot image on another area, then it should just call `dfu_target_done()` and initialize the new target with a new call to `dfu_target_init()`

Reply:
I dropped the whole sentence.
This is more clearly expressed on the section above it, in `Common application flow`